### PR TITLE
Add Tor plugin

### PR DESCRIPTION
**Pull request by ykray**
Adds menu item to manage system-wide Tor connection.

image

image

## Features
- Enable Tor
- Disable Tor
- Get new Tor identity

### CODE REVIEW

File: Network/tor.2s.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # * - UPDATE `sudoers` (to allow password-less network management)
 #     (1) sudo visudo
 #     (2) Append the following line at the end of the file:
-#             %admin          ALL = NOPASSWD: /usr/sbin/networksetup,killall
+#             %admin          ALL = NOPASSWD: /usr/sbin/networksetup,/usr/bin/killall
```
Comment (ykray) [inline Network/tor.2s.sh:19]: fixed `killall` path

File: Network/tor.2s.sh [hunk 2]
```diff
@@ -15,7 +15,7 @@
...
```
Comment (ykray) [inline Network/tor.2s.sh:18]: fixed `killall` path

File: Network/tor.2s.sh [hunk 3]
```diff
@@ -27,13 +27,6 @@ readonly icon_enabled="iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAEC2lDQ1BHZ
...
 
 enable_tor() {
-  # Keep-alive: update existing `sudo` timestamp until finished
-  while true; do
-    sudo -n true
-    sleep 60
-    kill -0 "$$" || exit
-  done 2>/dev/null &
-
```
Comment (ykray) [inline Network/tor.2s.sh]: removed since unnecessary with the `sudoers` file

**Comment by sprak3000** [comment]
@ykray 

You need to run `chmod +x` on the file.

**Comment by ykray** [comment]
done.

**Review by sprak3000 — APPROVED** [review]

**Pull request merged**